src/components/navbar-item/navbar-item.scss

```diff
@@ -0,0 +1,85 @@
+@import "../common/_base";
+
+:host {
+  @include visibility(inline-flex);
+  --gx-navbar-item-icon-size-big: 24px;
+  --gx-navbar-item-icon-size-small: 18px;
+
+  font-size: inherit;
+  font-weight: inherit;
+  outline: none;
+
+  @media screen and (max-width: $gx-xsmall-breakpoint) {
+    flex: 1;
+    justify-content: center;
+  }
+
+  button {
+    appearance: none;
+    border: 0;
+    background-color: transparent;
+    font: inherit;
+  }
+
+  .item {
+    flex: 1;
+    align-items: center;
+    cursor: pointer;
+    display: inline-flex;
+    color: inherit;
+    text-decoration: none;
+    padding: 3px 15px;
+    margin: 0 2px;
+    outline: none;
+
+    &--active {
+      background-color: #888;
```

Comment:
I think it will be great to turn the background-color from fixed color to customizable.

Something like:
```SCSS
  --gx-navbar-item-active-background-color: rgba(255, 255, 255, 0.6);
  --gx-navbar-item-active-text-color: rgba(0, 0, 0, 0.5);

  &--active {
     background-color: var(--gx-navbar-item-active-background-color);
     color: var(--gx-navbar-item-active-text-color);
  }
```

Reply:
You are 100% right. Will address this.